CHANGELOG.md

```diff
@@ -71,7 +71,8 @@ Here is an overview of all new **experimental** features:
 - **General**: Prevent a panic that might occur while refreshing a scaler cache ([#4092](https://github.com/kedacore/keda/issues/4092))
 - **Azure Service Bus Scaler:** Use correct auth flows with pod identity ([#4026](https://github.com/kedacore/keda/issues/4026)|[#4123](https://github.com/kedacore/keda/issues/4123))
 - **CPU Memory Scaler** Store forgotten logger ([#4022](https://github.com/kedacore/keda/issues/4022))
-  **Kafka Scaler**: Support 0 in activationLagThreshold configuration ([#4137](https://github.com/kedacore/keda/issues/4137))
+- **Kafka Scaler**: Support 0 in activationLagThreshold configuration ([#4137](https://github.com/kedacore/keda/issues/4137))
```

Comment:
please sort alphabetically

Reply:
For the lines you marked I just add a hyphen for one of the [previous contributions](https://github.com/kedacore/keda/commit/15805c99fdb5da242e8725777a03a479459af79f).  Or you mean to swap
```
- **Kafka Scaler**: Support 0 ...
- **Kafka Scaler**: Return ...
```

this lines?

Reply:
I swapped the lines. Should be fine now.